src/main/java/CarRacing/CarRacing.java

```diff
@@ -0,0 +1,55 @@
+package CarRacing;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class CarRacing {
+
+    public static final int BOUND = 10;
+    public static final int MIN_MOVE_VALUE = 4;
+
+    private static final Random random = new Random();
+
+    public boolean shouldMove(int randomValue) {
+        return randomValue >= MIN_MOVE_VALUE;
+    }
+
+    public Map<Integer, Integer> race(int carNums, int moveNums) {
+        Map<Integer, Integer> carsMap = initCarsMap(carNums);
+        repeatMove(carNums, moveNums, carsMap);
+        return carsMap;
+    }
+
+    private void repeatMove(int carNums, int moveNums, Map<Integer, Integer> carsMap) {
+        for(int k=1; k<= moveNums; k++) {
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
IDE로 IntelliJ IDEA를 사용 중이시라면,  자동 포맷팅 단축키(Windows에서는 `Ctrl + Alt + L`, Mac에서는 `Command + Option + L`)를 활용해보시면 좋을 것 같아요.
```suggestion
        for (int k = 1; k <= moveNums; k++) {
```

Reply:
자동 포맷팅 Command + Option + L 기억하도록 하겠습니다. 감사합니다.